### PR TITLE
feat(cdn/domain): add service_area argument support

### LINKED ISSUE #1466: [CDN] Add service_area support for cdn domain
Adds `service_area` parameter support, which is required for international account as it's defualt to `mainland_china`

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1466 

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccCdnDomain_basic'
...
=== RUN  TestAccCdnDomain_basic
--- PASS: TestAccCdnDomain_basic (436.87s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       436.907s
```

### CODE REVIEW

File: docs/resources/cdn_domain.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ The following arguments are supported:
 * `sources` - (Required, List, ForceNew) An array of one or more objects specifies the domain name of the origin server.
   The sources object structure is documented below.
 
+* `service_area` - (Optional, String, ForceNew) The area covered by the acceleration service. Valid values are
```
Comment (chengxiangdong) [inline docs/resources/cdn_domain.md:39]: I saw on the console that `service_area`  is a required field.

Comment (niuzhenguo) [reply docs/resources/cdn_domain.md:39]: it defaults to mainland_china... so it works before without this parameter :P
make it required will break current customers configuration.

Comment (chengxiangdong) [reply docs/resources/cdn_domain.md:39]: I see.

**Comment by chengxiangdong** [comment]
LGTM

**Comment by Jason-Zhang9309** [comment]
LGTM

**Pull request merged**